Automattic/wp-calypso
# Automattic for Agencies: Update favicons for different devices & meta tags on all A4A environments

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/199
Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/240

## Proposed Changes

This PR: 

- Updates the meta tags(description & image) on all the A4A environments.
- Updates Favicons for different devices 

## Testing Instructions

NOTE: We can test the meta image tag only in staging or production once this PR is deployed. 

- Open the Calyso live link for A4A.
- Copy the URL > Paste it in any Slack chat > Wait for a second > Hit Enter when it shows the preview and verify it looks like below. 

<img width="425" alt="Screenshot 2024-04-08 at 6 11 17 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/5e83c384-2e50-4156-aa41-cf2f2cb78e35">


## Pre-merge Checklist
